scss/components/sign-in-page.scss

```diff
@@ -0,0 +1,29 @@
+.not-authed {
+  min-height: 380px;
+  min-width: 476px;
+  padding: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  flex-direction: column;
+  position: relative;
+}
+
+.not-authed button {
+  border-radius: 25px;
+  color: #1da1f2;
+  display: flex;
+  align-items: center;
+}
```

Comment:
Let's eliminate the border with

```scss
border: none;
outline: none;
```

Let's make the background white 
```scss
background-color: white;
color: $twitter-blue;
```

The twitter image (which should be an svg) should be the same height as the text.

The font-weight should be bold.

There should be more padding around the right and left sides so that the button text does not extend into the semicircular part of the button.

![image](https://user-images.githubusercontent.com/6589960/100023398-e8514180-2db2-11eb-96a0-cdb3a8f2a0c2.png)

Reply:
Removed border and outline, reset background color, added twitter-blue variable, replaced twitter logo png file with svg file. Reset button padding. Moved img inline styles into scss file. 
![Screen Shot 2020-11-24 at 3 15 04 PM](https://user-images.githubusercontent.com/16155615/100146607-2e1c1180-2e68-11eb-9af1-7f5f1569c265.png)